locale/ms_MY/emails.po

```diff
@@ -120,6 +120,102 @@ msgstr ""
 "Terima kasih, <br />\n"
 "{$signature}"
 
+#, fuzzy
+msgid "emails.userValidateContext.body"
```

Comment:
I think these additions are unintentional.

Reply:
Yes, these got into during the rebase, my mistake. Fixed it . Also update the few more locale files where the missing locales(`emails.passwordReset.subject` and `emails.passwordReset.body`) added after the rebase .